pkg/apis/core/validation/validation_test.go

```diff
@@ -18693,6 +18693,22 @@ func TestValidateTopologySpreadConstraints(t *testing.T) {
 			errtype:  field.ErrorTypeInvalid,
 			errfield: "maxSkew",
 		},
+		{
+			name: "invalid minDomains",
```

Comment:
Also test a valid minDomains :)

Reply:
Sure. Added to "all required fields ok" test case.